modules/administration/pages/ssl-certs.adoc

```diff
@@ -294,3 +305,59 @@ To complete the process, you need to update the database with this command:
 ----
 
 If you use bootstrap, remember to also update your bootstrap scripts to reflect the new certificate information.
+
+
+
+== Replace Existing CA and Server Certificates
+
+You can replace working certificates on an existing {productname} installation with a new third party certificate.
+In this case, you can replace the old CA certificate RPM with the new one, and update the database.
+
+
+
+.Procedure: Replacing Existing Certificates
+
+. On the {productname} Server, at the command prompt, move the old certificate directory to a new location:
```

Comment:
This is correct, but confusing (old - new - old) and in the end, you call the new directory old-something ;)
Suggestion (and remove the next lines):

```suggestion
. On the {productname} Server, at the command prompt, move the old certificate directory to a backup location:

----
mv /root/ssl-build /root/backup-ssl-build
----
```

I did not check whether `/root/old-ssl-build` occurs later on.